index.html

```diff
@@ -52,4 +52,184 @@ <h1 class="intro height">I’m Kartikay Glad to see you!</h1>
       </ul>
     </div>
   </section>
+  <section>
+    <div class="grid-container">
+      <div class="img-container">
+        <img class="work-image" src="./Images/tonic-one.png" alt="work-Image" />
+      </div>
+      <div class="card">
+        <div class="card-element">
+          <h2 class="card-title">Tonic</h2>
+          <div class="card-section">
+            <p class="para-one">CANOPY</p>
+            <i class="fa fa-circle"></i>
+            <p class="para-two">Back End Dev</p>
+            <i class="fa fa-circle"></i>
+            <p class="para-two">2015</p>
+          </div>
+          <p class="card-body">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="tech-stack">
+            <button class="tech-items" type="button">html</button>
+            <button class="tech-items" type="button">css</button>
+            <button class="tech-items" type="button">javascript</button>
+          </div>
+          <a class="card-button" href="#">See Project</a>
+        </div>
+      </div>
+      <div class="img-container multipost">
+        <img class="work-image" src="./Images/Multiple-one.png" alt="work-Image" />
+      </div>
+      <div class="card multipost">
+        <div class="card-element">
+          <h2 class="card-title">
+            Multi-Post Stories
+          </h2>
+          <div class="card-section">
+            <p class="para-one">CANOPY</p>
+            <i class="fa fa-circle"></i>
+            <p class="para-two">Back End Dev</p>
+            <i class="fa fa-circle"></i>
+            <p class="para-two">2015</p>
+          </div>
+          <p class="card-body">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="tech-stack">
+            <button class="tech-items" type="button">html</button>
+            <button class="tech-items" type="button">css</button>
+            <button class="tech-items" type="button">javascript</button>
+          </div>
```

Comment:
- [ ] Great job on implementing this, but I still can think that you can do better, by the way, all technologies should have been wrapped in a `ul` tag and each technology in a `li`, please change this one too

Reply:
Thanks for the suggestion. I have considered your suggestion. In my case, I have designed it for a mobile version i.e 375 px and for that the design is working fine. I will make the changes as you suggested further in the Desktop version.

Reply:
This issue is not resolved

Reply:
@BenMukebo @Kingstalux 
Thanks for the suggestion. I have considered your suggestion. In my case, I have designed it for a mobile version i.e 375 px and for that the design is working fine. Moreover, this was not a mandatory requirement as per the project desription.
![Review2](https://user-images.githubusercontent.com/52527576/153697676-191f9908-8317-4efa-8224-af60f34368a4.png)

Reply:
Hello @thenarain, for best sematic practices, any lists should make use of the `ul` and `li` tags, you can place your `buttons` inside `li` tags and then place all the `li` tags inside a `ul` tag and you will be good to go.

Reply:
Thanks for the valuable suggestion. I have implemented all the changes.